plume-file-web-upload-jersey/README.md

```diff
@@ -14,21 +14,44 @@ Setup
   <artifactId>plume-file-web-upload-jersey</artifactId>
 </dependency>
 ```
+2. Create the webservice file with the injected dependency `FileMimeTypeDetector` (it comes from Plume file Core)
+
```

Comment:
Il faudrait ajouter un point 3 :
Choose your upload option between:
- Jersey Multipart: easier to setup, but offers lower performance
- Apache FileUpload: more difficult to setup, so it offers the best performance with no overhead

(avec des liens vers chacune des parties)